src/Associations.cc

```diff
@@ -537,127 +536,127 @@ void Associations::CollectMCStars(int realization)
 //
 //      MeasuredStarIterator msII;
 //      for(msII=catalog.begin();msII!=catalog.end();msII++)
-//      	{
-//      	  MeasuredStar& ms = **msII;
-//      	  cout << " *** >>> "
-//      	       << ms.GetMCMeas() << " "
-//      	       << ms.GetMCTruth() << endl;
-//      	}
+//          {
+//            MeasuredStar& ms = **msII;
+//            cout << " *** >>> "
+//                 << ms.GetMCMeas() << " "
+//                 << ms.GetMCTruth() << endl;
+//          }
```

Comment:
This can probably go once DM-5205 is done, but I'd rather not remove things until then.